config/routes.rb

```diff
@@ -1,9 +1,8 @@
 Rails.application.routes.draw do
   resources :status, only: [:index]
-  resources :proceeding_types, only: [:index]
 
   namespace 'v1' do
-    resources :status, only: [:index]
```

Comment:
Not sure why you deleted this. Wont status only be able to be accessed from the base url now?

Reply:
status is not related to versioning which is / v1, it should be outside on root level.
why do you think status should be like / v1/status?

Reply:
Ok that sounds reasonable. Thanks for clarifying